searxng_extra/standalone_searx.py

```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env python
-"""Script to run searx from terminal.
+# lint: pylint
+
+# SPDX-License-Identifier: AGPL-3.0-or-later
+# (C) Copyright Contributors to the SearXNG project.
+# (C) Copyright Contributors to the searx project (2014 - 2021)
```

Comment:
I have changed this line: I'm not the original author of this file.
See https://github.com/searx/searx/pull/1591